src/main/scala/codacy/codesniffer/docsgen/Generator.scala

```diff
@@ -0,0 +1,65 @@
+package codacy.codesniffer.docsgen
+
+import better.files.File
+import codacy.codesniffer.docsgen.parsers._
+import com.codacy.plugins.api.results.Tool
+import play.api.libs.json.{Json, Writes}
+
+import scala.collection.parallel.ForkJoinTaskSupport
+import scala.collection.parallel.immutable.ParSeq
+
+class Generator() {
+
+  private[this] val toolName = Tool.Name("PHP Code Sniffer")
+  private[this] val toolVersion = None
```

Comment:
Add version:
* Read from a files in the home of the projects and also use it for docker build
* When generating write it in patterns.json and then when building docker read from patterns.json